src/liboslexec/backendllvm.cpp

```diff
@@ -364,7 +364,11 @@ BackendLLVM::addCUDAVariable(const std::string& name, int size, int alignment,
 
     OSL_DASSERT (g_var && "Unable to create GlobalVariable");
 
+#if OSL_LLVM_VERSION >= 100
+    g_var->setAlignment  (llvm::MaybeAlign(alignment));
```

Comment:
Aside: what does that mean?

Reply:
No idea! It seems like an odd name to me too. But this does work and passes renderer tests.

Reply:
From the LLVM source:
```
// - Align represents an alignment in bytes, it is always set and always a valid
// power of two, its minimum value is 1 which means no alignment requirements.
//
// - MaybeAlign is an optional type, it may be undefined or set. When it's set
// you can get the underlying Align type by using the getValue() method.
```